### DISCUSSION TITLE
Jupyter book: Show content of a script file inside a book chapter

### DISCUSSION THREAD

**Discussion by jjkoehorst** [executablebooks/meta#818] [Q&A]
I am trying to maintain all the code in one location and not in the book and in a script folder.
Is it possible to show the content of an example.py within a page?

This makes it easier to maintain code changes and automatically update the book accordingly.

The code does not have to be executed.

**Comment by choldgraf** [comment]
Yes! Try the `literalinclude` directive:

````
```{literalinclude} ../example.py
```
````

Does that work?

**Reply by jjkoehorst** [reply]
Thanks for the fast reply! It works perfectly. 
I just did a quick search with this in https://jupyterbook.org/en/stable/search.html?q=literalinclude but there is no documentation for it?
